src/components/share-overlay/share-overlay.js

```diff
@@ -476,6 +476,30 @@ class ShareOverlay extends Component {
     this.props.setIsModal(true);
   }
 
+  /**
+   * update url with params and values
+   *
+   * @param {string} url - the share url
+   * @param {string} key - the param name
+   * @param {string} value - the param value
+   * @returns {string} - share url with the query parameters
+   * @memberof ShareOverlay
+   */
+  _updateUrlParams(url: string, key: string, value: number): string {
+    try {
+      const urlObj = new URL(url);
+      const searchParams = urlObj.searchParams;
+      if (searchParams.has(key)) {
+        searchParams.set(key, value);
+      } else {
+        searchParams.append(key, value);
+      }
+      return urlObj.toString();
+    } catch (e) {
+      this.props.logger.log(e);
```

Comment:
if you return nothing, the function will return undefined.
what about returning url ? or an empty string ?

Reply:
done